src/main/java/org/betterx/betterend/entity/SilkMothEntity.java

```diff
@@ -76,11 +81,12 @@ public void setHive(Level world, BlockPos hive) {
 
     @Override
     public boolean canBeLeashed(Player player) {
-        return false;
+        return true;
```

Comment:
Agnostic to this change, though I did validate that it's working:

![2022-10-30_13 44 42](https://user-images.githubusercontent.com/66568922/198893234-54687c04-f928-4a3e-b053-4e3ac1bec486.png)

 I didn't validate whether they'll follow you if you're holding food, and I'm actually not at all sure if a silk touched hive can be used to transport silk moths, so having this as a method to move your moths around seems like it would be a nice QoL change.